### PR TITLE
ID-498 [Feat] Adds biometrics documentation

### PR DESCRIPTION
**Pull request by tonytlwu**
Ref. https://weboo.atlassian.net/browse/ID-498

---

# Biometrics

_Biometrics are only available in native apps on supported devices._

### Check biometrics availability

Use the `Fliplet.User.Biometrics.isAvailable()` method to check whether biometrics are available on the device. The promise is rejected when biometrics are not available.

The available types are:

- `face` iOS and Android
- `finger` Android only
- `touch` iOS only

```js
Fliplet.User.Biometrics.isAvailable().then(function (type) {
  // Biometrics are available
}, function (err) {
  // Biometrics are not available
  console.error(err);
});
```

### Ask the user for biometric verification

Use the `Fliplet.User.Biometrics.verify()` method to verify the user using fingerprint or facial recognition.

```js
Fliplet.User.Biometrics.verify({
  title: 'Unlock your app',
  description: '' // (optional)
}).then(function () {
  // Verification passed
}, function (err) {
  // The popup is dismissed because the user canceled
  console.error(err);
});
```

### CODE REVIEW

File: docs/API/core/biometrics.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Biometrics
+
+_Biometrics are only available in native apps on supporting devices._
+
+### Check biometrics availability
```
Comment (squallstar) [inline docs/API/core/biometrics.md:5]: Why isn't this a `##` H2 heading instead?

File: docs/API/core/biometrics.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Use the `Fliplet.User.Biometrics.isAvailable()` method to checks whether biometrics are available on the device. The promise is rejected when biometrics are not available.
```
Comment (squallstar) [inline docs/API/core/biometrics.md:7]: "check", not "checks"

File: docs/API/core/biometrics.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The avilable types are:
```
Comment (squallstar) [inline docs/API/core/biometrics.md:9]: typo in "available"

**Review by squallstar — CHANGES_REQUESTED** [review]

File: docs/API/core/biometrics.md (hunk 2 above)
Comment (tonytlwu) [reply docs/API/core/biometrics.md]: Corrected

File: docs/API/core/biometrics.md (hunk 3 above)
Comment (tonytlwu) [reply docs/API/core/biometrics.md]: Corrected

File: docs/API/core/biometrics.md (hunk 1 above)
Comment (tonytlwu) [reply docs/API/core/biometrics.md:5]: Very good question. I'm guessing you did this for stylistic reasons.

https://developers.fliplet.com/API/core/apps.html
https://developers.fliplet.com/API/core/cache.html
https://developers.fliplet.com/API/core/encode.html
https://developers.fliplet.com/API/core/environment.html

**Comment by tonytlwu** [comment]
@squallstar The PR is ready for review again.

**Review by squallstar — APPROVED** [review]

**Pull request merged**